### ISSUE TITLE
Some questions in testing

### ISSUE THREAD

**Issue by csp2002** [med-air/3DSAM-adapter#42]
Thanks for your great work! I have a few questions when I run the UNETR++ using your code.

1. During the testing, I find that in some volumes the NSD and Dice are both 0. Have you met this in your testing? I guess it is because the model failed to segment the tumor?
2. In the following code in test.py, the img.shape=(1,1,187,246,246), the first masks.shape=(1,2,187,246,246). The original size of this volume is (671,512,512) and so the second masks.shape=(1,2,671,512,512). Could you explain more about how the data-preprocessing changes the size from (671,512,512) to (187,246,246)? Also, I am not so sure that simply using F.interpolate to change the size back makes sense because the model only seems to only predict that (187,246,246). Could you explain more in this post-processing?  Thank you!
masks = sliding_window_inference(
                img,
                roi_size=args.rand_crop_size,
                sw_batch_size=2,
                predictor=seg_net,
                mode=args.infer_mode,
                overlap=args.overlap,
            ) 
masks = F.interpolate(masks, size=seg.shape[1:], mode="trilinear")
